common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3468,6 +3468,10 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "          (Use with property hive.server2.custom.authentication.class)\n" +
         "  PAM: Pluggable authentication module\n" +
         "  NOSASL:  Raw transport"),
+    HIVE_SERVER2_TRUST_DOMAIN("hive.server2.trust.domain", "",
```

Comment:
Maybe we should call this trusted.domain (to be consistent with code)

Reply:
Done.